newm-server-portal/src/main/kotlin/io/projectnewm/server/portal/model/Song.kt

```diff
@@ -0,0 +1,19 @@
+package io.projectnewm.server.portal.model
+
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.serializers.LocalDateIso8601Serializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Song(
+    @SerialName("id") val id: Int,
+    @SerialName("name") val name: String,
+    @SerialName("description") val description: String,
+    @SerialName("genre") val genre: String,
+    @SerialName("releaseDate") val releaseDate: LocalDate,
+    @SerialName("albumImageUrl") val albumImageUrl: String,
+    @SerialName("extraInfo") val extraInfo: String,
+    @SerialName("userRole") val userRole: Role,
```

Comment:
@HenryQuillin What is this used for? Not sure why but seems out of place for a Song class

Reply:
@j-mateo Which property? userRole? I added that property because one of the inputs in the song upload form is User Role (singer, producer, etc). However, since we are using a graph database it may be unnecessary -- we could have predicates (edges) that define the relationship between the user object and the song object.

Reply:
I removed it for now. It sounds to me like the role can come from the User profile.